consensus/config/src/committee.rs

```diff
@@ -21,7 +19,7 @@ pub type Stake = u64;
 
 /// Committee is the set of authorities that participate in the consensus protocol for this epoch.
 /// Its configuration is stored and computed on chain.
-#[derive(Serialize, Deserialize)]
+#[derive(Clone, Serialize, Deserialize)]
```

Comment:
Similar to bellow, why we need to Clone Committee? I assume we can just get away by passing around the `Arc<Committee>`